src/geometry/CCPACSTransformation.cpp

```diff
@@ -118,6 +125,8 @@ ECPACSTranslationType CCPACSTransformation::getTranslationType() const
 
 CTiglTransformation CCPACSTransformation::getTransformationMatrix() const
 {
+    if (invalidated)
+        updateMatrix();
```

Comment:
Please use curly braces even in single lined if statements. This is to prevent errors, when adding another line to the if-statement.

Reply:
I don't think this is a common source of errors. And by adding braces you litter the code by two more signs and an additional line.
But I can live with your opinion.

Reply:
https://blog.codecentric.de/en/2014/02/curly-braces/

Reply:
i really like the second article, it clearly highlights that the missing braces are only a small part of the problem. The code in general is quite tough to read already by the assignments inside the ifs, gotos, variable declarations at the function block begin and error checking.

Anyhow, I added the braces, so you can sleep better ;)

Reply:
I thought it would be in the style guide, which actually isn't. So not your fault.

Sure, this is just a small part of the problem. I would say, without braces would be okay, if directly after the if statement (without newline).

Thanks for taking care of my sleep 😀